### PR TITLE
150 create new board cogip board

### LINKED ISSUE #150: Create new board cogip-board
Create new board for 2024 robot.

### PR DESCRIPTION
**Pull request by gdoffe**
Create new cogip-board with STM32G474RE for FDCAN support.

Close #150

### CODE REVIEW

File: boards/cogip-board/board.c [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2023 COGIP Robotics association
```
Comment (ecourtois) [inline boards/cogip-board/board.c:2]: 2024

File: boards/cogip-board/include/board.h [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_cogip-board COGIP 2023 Pegasus board
+ * @ingroup     boards_cogip-board
+ * @brief       Support for the COGIP 2023 Pegasus board
```
Comment (ecourtois) [inline boards/cogip-board/include/board.h:12]: 2024

Pegasus is the platform name, the board name.

File: boards/cogip-board/include/periph_conf.h [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+ * Copyright (C) 2024 COGIP Robotics association
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_cogip-board
+ * @brief       Support for the COGIP 2023 Pegasus board
```
Comment (ecourtois) [inline boards/cogip-board/include/periph_conf.h:11]: pegasus - 2024

File: boards/cogip-board/include/periph_conf.h [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+ * @{
+ *
+ * @file
+ * @name        Peripheral MCU configuration for the cogip-board-ng board
```
Comment (ecourtois) [inline boards/cogip-board/include/periph_conf.h:15]: cogip-board-ng

**Review by ecourtois — APPROVED** [review]
Just a few remarks in the comments, but I already approuve.

File: boards/cogip-board/include/periph_conf.h (hunk 4 above)
Comment (gdoffe) [reply boards/cogip-board/include/periph_conf.h:15]: done

File: boards/cogip-board/include/periph_conf.h (hunk 3 above)
Comment (gdoffe) [reply boards/cogip-board/include/periph_conf.h:11]: done

File: boards/cogip-board/include/board.h (hunk 2 above)
Comment (gdoffe) [reply boards/cogip-board/include/board.h:12]: done

File: boards/cogip-board/board.c (hunk 1 above)
Comment (gdoffe) [reply boards/cogip-board/board.c:2]: done

**Pull request merged**